helpers-assets/js/translation-discussion.js

```diff
@@ -25,7 +25,8 @@ jQuery( function( $ ) {
 			post: $commentform.attr('id').split( '-' )[ 1 ],
 			meta: {
 				translation_id : $commentform.find('input[name=translation_id]').val(),
-				locale         : $commentform.find('input[name=comment_locale]').val()
+				locale         : $commentform.find('input[name=comment_locale]').val(),
+				comment_topic  : $commentform.find('select[name=comment_topic]').val(),
```

Comment:
I'm a bit worried that someone could add arbitrary text here. Could you check which options we have to use a list of allowed entries to check against? Probably something in the REST API.